cloudant-client/src/main/java/com/cloudant/client/api/query/QueryBuilder.java

```diff
@@ -24,36 +24,144 @@
     private Sort[] sort;
     private Long limit;
     private Long skip;
+    private String bookmark;
+    private boolean update = true;
+    private boolean stable;
+    private boolean executionStats;
+    private String[] useIndex;
 
+    /**
+     * Construct a {@code QueryBuilder} object for use with {@link com.cloudant.client.api.Database#query(String, Class)}
+     * @param selector JSON object describing criteria used to select documents.
+     * @see <a
```

Comment:
I think perhaps there should be an overall usage example in the class javadoc which perhaps is the right place to show it being used with an `executeRequest` as well.

Reply:
See e62187e